<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighScoresDialog</class>
 <widget class="QDialog" name="HighScoresDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>High scores</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="cbScoreType"/>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>HighScoresDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>HighScoresDialog</receiver>
   <slot>clearScores()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbScoreType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>HighScoresDialog</receiver>
   <slot>matchTypeIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clearScores()</slot>
  <slot>matchTypeIndexChanged(int)</slot>
 </slots>
</ui>
